projects/canopy-test-app/src/polyfills.ts

```diff
@@ -21,7 +21,7 @@
 /** IE10 and IE11 requires the following for NgClass support on SVG elements */
 // import 'classlist.js';  // Run `npm install --save classlist.js`.
 
-import 'core-js/es7/array';
+// import 'core-js/es7/array';
```

Comment:
Could this and the classlist line be removed at all?

Reply:
Updated 👍